### ISSUE TITLE
CONTRIBUTORS and Contributors folders need to be deleted.

### ISSUE THREAD

**Issue by kwurst** [jlord/patchwork#27932]
@jlord:

`CONTRIBUTORS` and `Contributors` folders have been created. 

When cloned on a Mac they overwrite the `contributors` folder with a `CONTRIBUTORS` folder.

- If you create the `add-username.txt` file in your `add-username` branch in the `CONTRIBUTORS` folder and commit, `reporobot` will not find it.
- If you rename `CONTRIBUTORS` to `contributors`, then add the file and commit, `reporobot` will not find it.
- If you delete `CONTRIBUTORS`, create `contributors`, then add the file and commit, `reporobot` *will* find it and add the picture. BUT, when you submit the pull request, you get a complaint about too many files. https://github.com/jlord/patchwork/pull/27931

The `CONTRIBUTORS` and `Contributors` folders need to be deleted. And there must be some way to prevent others from recreating them, or to prevent `reporobot` from merging them in a pull request.

**Comment by FrankwaP** [comment]
Yes, these 3 case-different folders are a PITA for Windows and Mac users, since the filenames/foldernames are case insensitive for these two OS.

I did this:
1. delete the `CONTRIBUTORS` and `Contributors` folder
2. creating a `tmp` folder and store my `add-usernam.txt` in it
3. rename this `tmp` folder to `contributors`. I'm not so sure, but I think `git mv` works better than `mv` alone in this regard: with the first one I get `contributors/add-usernam.txt` in y commit, but with the latest I get `CONTRIBUTORS/add-usernam.txt`. I'm too much of a beginner to understand why...
4. `git status` to check the only things that will be committed is `contributors/add-usernam.txt` (no deletion of anything)
5. commit/push/pull-request

My pull-request has been accepted, and I now have my name on the webpage! 

... but I can't see my file in the `contributors` folder of jlord's depo ¯\_(ツ)_/¯

**Comment by jlord** [comment]
Hey!

Sorry, that folder is supposed to be getting periodically cleared so that it doesn't get so huge but that's apparently broken 😢 I'll open an issue for it on https://github.com/jlord/reporobot/issues/40.

**Comment by KateDay** [comment]
This is still an issue btw...

**Comment by hernandez87v** [comment]
anyone else stuck theres another "Patchwork" folder within patchwork that contains "contributors":

- Lesson 7 Branches Aren't Just For Birds
- cd Patchwork
- when you get to "$ git add <FILENAME>" put "git add contributors/add-yourUsername"

**Comment by MeghanBarritt** [comment]
The patchwork>contributors folder is not actually the solution anymore, at least on Windows. I have tried putting my txt file in all of the different 'contributors' folders, as well as deleting the nested contributors folder. The only time I manage to push at all was when I had the txt file in completely the wrong folder (the top level project folder), but I cannot replicate that and I was unable to make that merge. @jlord
